sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -1306,7 +1309,7 @@ class MLPRegressor(RegressorMixin, BaseMultilayerPerceptron):
         layer i + 1.
 
     n_iter_ : int
-        The number of iterations the solver has ran.
+        The number of iterations the solver has run.
```

Comment:
I think ending a sentence with "ran" or "run"` does not read nicely in english. How about:

```suggestion
        Number of iterations used by the solver.
```

Reply:
I intended only to correct the grammar, not to change the wording.

Since the attribute keeps a running count of iterations perhaps it should read, "... has run thus far". The phrase, "used by the solver" does not convey a running count.

If you want to keep it strictly technical, I might suggest, "The running count of solver iterations."

Reply:
Okay, let stick with what you have right now. We can adjust the wording if someone finds it confusing.